dotnet/samples/KernelSyntaxExamples/Example65_HandlebarsPlanner.cs

```diff
@@ -59,9 +68,20 @@ private static async Task RunSampleAsync(string goal, params string[] pluginDire
                 apiKey: apiKey)
             .Build();
 
-        if (pluginDirectoryNames[0] == DictionaryPlugin.PluginName)
+        if (pluginDirectoryNames[0] == BasicDictionaryPlugin.PluginName)
+        {
+            kernel.ImportPluginFromObject(new BasicDictionaryPlugin(), BasicDictionaryPlugin.PluginName);
+        }
+        else if (pluginDirectoryNames[0] == ComplexDictionaryPlugin.PluginName)
+        {
+            kernel.ImportPluginFromObject(new ComplexDictionaryPlugin(), ComplexDictionaryPlugin.PluginName);
+        }
```

Comment:
The naming of the two dictionary plugins isn't super obvious, something like the following would be clearer

1. `StringParamsDictionaryPlugin`
1. `ComplexParamsDictionaryPlugin`

Reply:
Done